### PR TITLE
Review workflow

### PR DESCRIPTION
**Pull request by liadomide**
Together with Paula we:
- reviewed the classes in package `bnm.recon.qc.model`
- merge latest changes from branch `DenisWorkflow`
- fixed the pegasus simple demo after recent file moves between folders

The review of the Python code is still work in progress, but we think that it is wise to merge in master the current state `559579c`

### CODE REVIEW

File: config/patient_run_config.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#TODO place these in a better form ?
+
+PEGASUS_HOME=/opt/pegasus-home
+echo ${PEGASUS_HOME}
+export PEGASUS_HOME
```
Comment (maedoc) [inline config/patient_run_config.sh]: it's more usual and compact to write one liners, `export FOO=bar`

File: bin/reconutils.py [hunk 2]
```diff
@@ -5,11 +5,12 @@
 import sys
 import glob
 import numpy as np
-import nibabel
+import nibabel as nbl
```
Comment (maedoc) [inline bin/reconutils.py]: for future, I prefer not to abbrev module names other than numpy (which is used in arith expr)

Comment (liadomide) [inline bin/reconutils.py]: I agree

**Comment by maedoc** [comment]
just curious about the LICENSE file, then we can merge

File: config/patient_run_config.sh (hunk 1 above)
Comment (liadomide) [inline config/patient_run_config.sh]: Noted

File: data/freesurfer_fsaverage/LICENSE [hunk 3]
```diff
@@ -0,0 +1,191 @@
+         FreeSurfer Software License Agreement ("Agreement")
```
Comment (maedoc) [inline data/freesurfer_fsaverage/LICENSE]: The title on that license uses the word Software, not data. I think they would fine with us using it as a test data set, but we ought to ask on the FS mailing list to be sure.

Comment (liadomide) [inline data/freesurfer_fsaverage/LICENSE]: I think the idea to ask the FS mailing list is a good one. Shall I do it, or do you want to ask?

I saw the  word Software in the title, but I consider the second paragraph in the license document, which seems to be saying that they refer by "Software" to both code and data. Did I misunderstood? 

> .... distributing or use of any software and/or data from FreeSurfer (collectively, the "Software")

**Comment by maedoc** [comment]
I sent a question to the FS list, will link reply when available

**Comment by maedoc** [comment]
Bruce Fischl, author of FS, says it would be fine. For reference

http://www.mail-archive.com/freesurfer@nmr.mgh.harvard.edu/msg50053.html

**Pull request merged**